wp/plugin/wp.ml

```diff
@@ -18,149 +18,158 @@ module Typ = Extension.Type
-          WP will check that the modified binary has no additional null \
-          dereferences in comparison with the original binary."
+    ~doc:{|Checks for inputs that would result in dereferencing a null value
+           during a memory read or write. In the case of a comparative
+           analysis, checks that the modified binary has no additional null
+           dereferences in comparison with the original binary.|}
 
 let compare_func_calls = Cmd.flag "compare-func-calls"
-    ~doc:"This flag is used for a comparative analysis. If set, WP will \
-          check that function calls should not occur in the modified binary \
-          if they have not occurred in the original binary."
+    ~doc:{|This flag is only used in a comparative analysis. Checks that
+           function calls do not occur in the modified binary if they have not
+           occurred in the original binary.|}
 
 let compare_post_reg_values = Cmd.parameter Typ.(list string) "compare-post-reg-values"
-    ~doc:"This flag is used for a comparatve analysis. If set, WP will \
-          compare the values stored in the specified registers at the end of \
-          the analyzed function's execution. For example, `RAX,RDI' compares \
-          the values of RAX and RDI at the end of execution. If unsure about \
-          which registers to compare, x86_64 architectures place their \
-          output in RAX, and ARM architectures place their output in R0."
+    ~doc:{|This flag is only used in a comparatve analysis. Compares the values
+           stored in the specified registers at the end of the function's
+           execution. For example, `RAX,RDI' compares the values of RAX and RDI
+           at the end of execution. If unsure about which registers to compare,
+           x86_64 architectures place their output in RAX, and ARM
+           architectures place their output in R0.|}
 
 (* Options. *)
 
 let inline = Cmd.parameter Typ.(some string) "inline"
-    ~doc:"Function calls to inline at a function call site as specified by a \
-          POSIX regular expression on the name of the target function. If \
-          not inlined, function summaries are used at function call time. To \
-          inline everything, set to `.*'. For example, `foo|bar' will inline \
-          the functions foo and bar."
+    ~doc:{|Functions specified by the provided POSIX regular expression will be
+           inlined. When functions are not inlined, heuristic function
+           summaries are used at function call sites. For example, if you want
+           to inline the functions `foo' and `bar', you can write
+           `--inline=foo|bar'. To inline everything, use `--inline=.*' (not
+           generally recommended).|}
 
 let num_unroll = Cmd.parameter Typ.(some int) "num-unroll"
-    ~doc:"Amount of times to unroll each loop. By default, WP will unroll \
-          each loop 5 times."
+    ~doc:{|Replaces the default number of times to unroll each loop. WP will
```

Comment:
Instead of "Replaces the default number of times" maybe say "Specifies the number of times"